nasa-gibs/worldview
# wv-2387 Fix Compare Mode Swipe in Polar Projections

## Description
When using compare mode in polar projections, the point at which the A & B sides meet was rotating incorrectly such that the separation point between the 2 layers did not maintain a vertical orientation. This fix updates the clipping mask generation code so the vertical orientation (beneath the swipe bar) is maintained.

Fixes # WV-2387

## How To Test
- Checkout branch wv-2387-compare-rotate
- npm ci
- npm run watch
- Confirm that compare mode in Geographic mode works normally
- Change to polar projection & confirm that compare mode is behaving properly. Make sure to rotate the maps by using the buttons on the right side & also by pressing ALT on Windows or OPTION on MAC & clicking/dragging with your mouse.

A good layer to make the A/B comparison obvious is Sea Ice Concentration, 12km. Set the date between 2002-2011 to get the imagery.

@nasa-gibs/worldview
